lib/support/cli.js

```diff
@@ -466,6 +466,12 @@ export function parseCommandLine() {
       type: 'boolean',
       group: 'firefox'
     })
+    .option('firefox.geckoProfilerCustom', {
+      describe: 'Expose the start/stop commands for the gecko profiler',
+      default: false,
+      type: 'boolean',
+      group: 'firefox'
+    })
```

Comment:
`firefox.geckoProfilerCustom` name is a bit vague to me. How about making it something like `geckoProfilerRecordingType`, which can take an enum of either 'pageload' or 'custom'. and default it to 'pagelod'?

Reply:
Good idea, perhaps part of firefox.geckoProfilerParams? an optional cli flag to pass like `firefox.geckoProfilerParams.recordingType`

Reply:
(or even firefox.geckoProfiler.recordingType)

Reply:
I think `firefox.geckoProfilerParams` is different. The options under this scope are the parameters we pass inside the profiler start function as arguments. So I think it's better to separate this with them.

I like the `firefox.geckoProfiler.recordingType` idea! That way, it's clear that it's related to the `firefox.geckoProfile` as it's scoped that way.